portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,108 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@keyframes textShadow {
```

Comment:
Add a comment here to briefly explain what the `textShadow` do.

Reply:
Done.